test/test_core.py

```diff
@@ -7493,9 +7493,14 @@ def test_embind_dynamic_initialization(self):
     self.do_run_in_out_file_test('embind/test_dynamic_initialization.cpp')
 
   @no_wasm2js('wasm_bigint')
-  def test_embind_i64_val(self):
+  @parameterized({
+    '': ([],),
+    'safe_heap': (['-sSAFE_HEAP'],),
+  })
+  def test_embind_i64_val(self, extra_args):
     self.set_setting('WASM_BIGINT')
     self.emcc_args += ['-lembind']
+    self.emcc_args += extra_args
```

Comment:
Maybe just do this one line `self.emcc_args += ['-lembind'] extra_args`.

I think the more common pattern is to call the param just `args`